lib/scope.go

```diff
@@ -74,14 +97,16 @@ func (s *scope) Config() *config.Config {
 	return s.inst.cfg
 }
 
-// Context returns the context for this scope. Though this pattern is usually discouraged,
-// we're following http.Request's lead, as scope plays the same role. The lifetime of a
-// single scope matches the lifetime of the Context; this ownership is not long-lived
+// Context returns the context for this scope. Though this pattern is usually
+// discouraged, we're following http.Request's lead, as scope plays the same
+// role. The lifetime of a single scope matches the lifetime of the Context;
+// this ownership is not long-lived
```

Comment:
I've added some doc tweaks to this file. Code ruler should be set to 80 characters for comments